### ISSUE TITLE
NTR: LSM1-7 & LSM2-8 complex

### ISSUE THREAD

**Issue by bmeldal** [geneontology/go-ontology#14411]
Neither of these complexes have a defined function (yet). But they are highly conserved from yeast to humans to plants so I thought it might warrant composition-specific terms here.

1. 
Name: LSM1-7 complex
Def: A protein complex essential for P-body formation and deadenylation-dependent decapping of mRNA thereby regulating mRNA decay and subsequent degradation by the 5' to 3' pathway. 

xrefs:
GOC:bhm
IntAct:EBI-16419327
PMID:23221597, 23620288, 26764377

Relationships:
is_a GO:0043234 protein complex
part_of GO:0000932 P-body
capable_of_part_of GO:NEW positive regulation of GO:0000290 deadenylation-dependent decapping of nuclear-transcribed mRNA (https://github.com/geneontology/go-ontology/issues/14410)

2.
Name: LSM2-8 complex
Def: A protein complex that is part of the U6 snRNP spliceosome machinery. Stabilises U6 small nuclear RNA to ensure the efficiency and accuracy of constitutive and alternative splicing of selected pre-mRNAs depending on the environmental conditions.

xrefs:
GOC:bhm
IntAct:EBI-16419597
PMID:23221597, 23620288, 28482101

Relationships:
is_a GO:0043234 protein complex
part_of GO:0005634 nucleus
part_of GO:0005688 U6 snRNP
[or maybe GO:0005688 U6 snRNP has_part LSM2-8 complex? I always find this relationship confusing!]
capable_of_part_of GO:0000398 mRNA splicing, via spliceosome
capable_of GO:0017070 U6 snRNA binding

Thanks,
Birgit

**Comment by ValWood** [comment]
GO:0005688 U6 snRNP
is  probably equivalent to : LSM2-8 complex 

 it shouldn't be a parent of  Name: LSM1-7 complex-  because it is specifically the complex which binds to small nuclear RNA U6  and it is vey well known by this name.

**Comment by ValWood** [comment]
yes they seem to be the dame compositionally.

please make LSM2-8 complex a synonym of  U6 snRNP instead
--

**Comment by bmeldal** [comment]
But:
Def: A ribonucleoprotein complex that **contains small nuclear RNA U6**. The LSM2-8 does not contain the RNA, acc to the papers we read.

> it shouldn't be a parent of Name: LSM1-7 complex- because it is specifically the complex which binds to small nuclear RNA U6 and it is vey well known by this name.

I don't think I suggested that, the 2 complexes are different.

**Comment by ValWood** [comment]
but doesn't it bind to the snRNA to exert its function?
https://en.wikipedia.org/wiki/U6_spliceosomal_RNA

@krchristie  will know more.

**Comment by ValWood** [comment]
I see its already assigned to @krchristie

**Comment by bmeldal** [comment]
binding to snRNA doesn't necessarily mean it's part of it.

**Comment by krchristie** [comment]
> GO:0005688 U6 snRNP
> is probably equivalent to : LSM2-8 complex

No, these are not equivalent. LSM2-8 complex is specifically the heptameric LSM ring, which is part_of the U6 snRNP.

**Comment by krchristie** [comment]
Well, it's definitely accurate to say that the U6 snRNP always has_part the LSM2-8 ring. I'd have to check if we can say that the LSM2-8 ring is always part_of the U6 snRNP...

**Comment by krchristie** [comment]
Seems if we are going to have the Lsm1-7 and Lsm2-8 complexes, we should also have the Sm protein complex. Perhaps we should also have a parent term called something like "Sm-like protein family complex" as a grouping term for all of them. There looks to be precedent in the ontology for "____ family complex" terms.

**Tharun S**. Roles of eukaryotic Lsm proteins in the regulation of mRNA function.
Int Rev Cell Mol Biol. 2009;272:149-89. doi: 10.1016/S1937-6448(08)01604-3.
Review. PMID: 19121818

> The Sm proteins are the founding members of the Sm-like protein family. There are seven Sm proteins that are conserved in all eukaryotes. They are B/B', D1, D2, D3, E, F, and G. These proteins assemble into a ring-shaped heteroheptamer on the U snRNAs of the major (U1, U2, U4, and U5 snRNAs) and minor (U11, U12, and U4atac snRNAs) spliceosomes forming the core of the corresponding snRNPs (Lerner and Steitz, 1979, 1981; Luhrmann et al, 1990; Will et al, 1999). The Sm proteins are required for the stability of the snRNAs and biogenesis of the snRNPs and therefore are very important in pre-mRNA splicing (Will and Luhrmann, 2001). All eukaryotes have an Lsm protein that is homologous to each of the six Sm proteins D1, D2, D3, E, F, and G named Lsm2, Lsm3, Lsm4, Lsm5, Lsm6, and Lsm7, respectively, and two more Lsm proteins, Lsm1 and Lsm8 that are weakly related to the Sm-B protein (Achsel et al, 1999; Salgado-Garrido et al, 1999; Seraphin, 1995).

**Comment by krchristie** [comment]
Regarding use of relationships with part_of or has_part, consider whether either of these statements is true, using Lsm2-8 complex as an example:

- part_of 
 The **Lsm2-8 complex** is **ALWAYS** _part_of_ the **U6 snRNP complex**
 if ALWAYS true, then you can use the _part_of_ relationship

- has_part 
 The **U6 snRNP complex** ALWAYS _has_part_ the **Lsm2-8 complex** 
 if ALWAYS true, then you can use the _has_part_ relationship


**Name: LSM1-7 complex**
Def: A **heteroheptameric protein complex composed of Lsm1, Lsm2, Lsm3, Lsm4, Lsm5, Lsm6, and Lsm7, or orthologs thereof, that complex selectively binds to mRNPs targeted for decay. It is ** essential for deadenylation-dependent decapping of mRNA and P-body formation thereby regulating mRNA decay and subsequent degradation by the 5' to 3' pathway. `[I think we need to include composition.]`

xrefs:
GOC:bhm
IntAct:EBI-16419327
PMID:23221597, 23620288, 26764377 `[not sure I'm keen to put three A. thaliana refs on defs for a highly conserved complex, would prefer to use the review I cited above.]`

Relationships:
is_a GO:0043234 protein complex
part_of GO:0000932 P-body `[I'm not sure we can say this complex is ALWAYS part_of the P-body. I kind of thought that it was involved in bringing mRNAs subject to decay TO the P-body, implying that it binds them elsewhere and helps to localize them there]`
capable_of_part_of GO:NEW positive regulation of GO:0000290 deadenylation-dependent decapping of nuclear-transcribed mRNA (#14410)

Also note that we already have a term for the **Lsm1-7-Pat 1 complex**. It seems like most of the functions you ascribe to the Lsm1-7 complex are when it is bound to Pat1, so I wonder if it would be more appropriate to put the description of biological role in mRNA decay and P-body formation into that def, and put only the RNA binding part in the def of the Lsm1-7 complex term.

**Name: LSM2-8 complex**
Def: A protein complex that is part of the U6 snRNP spliceosome machinery. `This is not all it does` Stabilises U6 small nuclear RNA to ensure the efficiency and accuracy of constitutive and alternative splicing of selected pre-mRNAs depending on the environmental conditions.
Revised Def: A **heteroheptameric protein complex composed of Lsm2, Lsm3, Lsm4, Lsm5, Lsm6,  Lsm7, and Lsm8, or orthologs thereof, that complex selectively binds to the U6 snRNA and also to 

xrefs:
GOC:bhm
IntAct:EBI-16419597
PMID:23221597, 23620288, 28482101 `[again, not sure I'm keen to put three A. thaliana refs on defs for a highly conserved complex, would prefer to use the review I cited above.]`

Relationships:
is_a GO:0043234 protein complex
part_of GO:0005634 nucleus
part_of GO:0005688 U6 snRNP
[or maybe GO:0005688 U6 snRNP has_part LSM2-8 complex? I always find this relationship confusing!]
capable_of_part_of GO:0000398 mRNA splicing, via spliceosome
capable_of GO:0017070 U6 snRNA binding

`saving now, but will come back and revise this comment`

**Comment by ValWood** [comment]
Ah OK, so how does  LSM2-8 complex differ from   "GO:0005688 U6 snRNP"?
I only have Lsm2-8 annotated to this.

**Comment by krchristie** [comment]
The Lsm2-8 complex is a heteroheptameric ring-shaped complex that does not contain the U6 snRNA.

It is also worth noting that the Lsm2-8 complex also binds to unspliced transcripts within the nucleus and is  involved in degradation, so I think it is entirely appropriate to have a term specifically for the Lsm2-8 ring/complex since it is present in multiple other complexes.

**Comment by bmeldal** [comment]
Thanks, Karen. I'm only work 2 days this week due to school holidays and they were jampacked! I'll get back to your questions above next week.

**Comment by krchristie** [comment]
Useful quote in thinking of a definition for grouping term **"Sm-like protein family complex"** 
> An interesting feature of the Sm-like proteins is their ability to assemble into complexes of multiple combinations of varied functions and RNA binding specificities (Beggs, 2005; Wilusz and Wilusz, 2005).

**Comment by krchristie** [comment]
Here's info on the Sm/Lsm ring present in the core U7 snRNP complex:

from: **Marzluff WF, Koreski KP**. Birth and Death of Histone mRNAs. Trends Genet. 2017 
Oct;33(10):745-759. doi: 10.1016/j.tig.2017.07.014. Epub 2017 Aug 31. Review. PMID:28867047

> U7 snRNA is a small (<70 nt) RNA containing a 2,2,7 trimethyl G cap. Like the spliceosomal snRNPs, U7 snRNP can exist as a core snRNP and a larger holo-snRNP which contains factors essential for processing. The core U7 snRNP consists of U7 snRNA bound to five Sm proteins, B, D3, E, F, and G, that are found in spliceosomal snRNAs, and to Lsm10 and Lsm11 which replace SmD1 and SmD2 [18,19]. Lsm11 is much larger than the other Sm proteins, 360 amino acids (aa) in mammals and 256 aa in Drosophila, and the N terminus of Lsm11 plays a crucial role in histone pre-mRNA processing [20].

**Comment by krchristie** [comment]
The Tharun review also discusses some other Lsm family complexes:
- **Sm complex** (SmB/B', SmD1, SmD2, SmD3, SmE, SmF, and SmG), binds to all the snRNAs EXCEPT U6
- **U8 snoRNA associated Lsm complex**, possibly only six membered (Lsm8, Lsm2, Lsm3, Lsm4, Lsm6, Lsm7 [no known homolog of Lsm5]
- **snR5 associated Lsm complex**, possibly only six membered ([no known homolog of Lsm 1 or Lsm8] but containing Lsm2, Lsm3, Lsm4, Lsm5, Lsm6, Lsm7

**Comment by krchristie** [comment]
@bmeldal - No worries about not getting to this till next week. I've been putting my thoughts into various comments, both suggested revisions to the two terms you proposed, as well as others that might be merited, perhaps with a structure like I've put below. Defs of these should focus on RNA binding function, especially for those that are present in more than one complex. Many of the processes you mentioned in your initially proposed defs more likely belong in the defs of the larger complexes with additional components that actually have those roles, e.g. "Lsm1-7-Pat1 complex" and "U6 snRNP".

-- Lsm family ring complex
--- Lsm1-7 complex
--- Lsm2-8 complex
--- Sm complex
--- U8 snoRNA associated Lsm complex
--- snR5 associated Lsm complex

**Comment by thomaspd** [comment]
I think we should have a discussion about functional classifications of complexes.  I would suggest that we ANNOTATE complexes with functions, complete with evidence code and reference, rather than defining their functions in the ontology.  The GO terms could reflect complex membership.

**Comment by bmeldal** [comment]
Thanks you @krchristie, here my comments:

In principle I agree with everything so let's get this finalised:

> Well, it's definitely accurate to say that the U6 snRNP always has_part the LSM2-8 ring. I'd have to check if we can say that the LSM2-8 ring is always part_of the U6 snRNP...

LSM2-8 is not always part of U6 snRNP:

PMID:22446625
Tang et al (2012). Telomerase RNA biogenesis involves sequential binding by Sm and Lsm complexes. Nature. 2012 Mar 25;484(7393):260-4. doi: 10.1038/nature10924.
> "Here we demonstrate that the canonical Sm ring and the **Lsm2–8 complex sequentially associate with fission yeast TER1**. The Sm ring binds to the TER1 precursor, stimulates spliceosomal cleavage and promotes the hypermethylation of the 5′-cap by Tgs1. Sm proteins are then replaced by the Lsm2–8 complex, which promotes the association with the catalytic subunit and protects the mature 3′-end of TER1 from exonucleolytic degradation."
Lots of experimental evidence.

> Seems if we are going to have the Lsm1-7 and Lsm2-8 complexes, we should also have the Sm protein complex. Perhaps we should also have a parent term called something like "Sm-like protein family complex" as a grouping term for all of them. There looks to be precedent in the ontology for "____ family complex" terms.

I think "Sm-like protein family complex" is a good idea as there are already a number of other Sm-X complexes in GO.

> Tharun S. Roles of eukaryotic Lsm proteins in the regulation of mRNA function.
Int Rev Cell Mol Biol. 2009;272:149-89. doi: 10.1016/S1937-6448(08)01604-3.
Review. PMID: 19121818

I have no access to the fulltext for this one but I have found similar statements elsewhere. 

Essentially, Sm-related complexes stablise snRNAs and provide the basis for bigger snRNP complexes. These occur in a manifold of combinations with fewer or more associated proteins and other complexes depending on cellular circumstances (see review PMID:27627834)
A case in point in PMID:28768202 (LSM1-7 and LSM2-8 both associate with Pat1B but are cytoplasmic- and nuclear-specific, resp.)
Also: Sm complexes don't assemble without the snRNA while LSM complexe assemble on their own. This may need to go in the def or a comment.

**Term requests review:**

Name: LSM1-7 complex
Def: Def: A protein complex essential for P-body formation and deadenylation-dependent decapping of mRNA thereby regulating mRNA decay and subsequent degradation by the 5' to 3' pathway.
Revised Def: A heteroheptameric, **cytoplasmic** protein complex composed of Lsm1, Lsm2, Lsm3, Lsm4, Lsm5, Lsm6, and Lsm7, or orthologs thereof, that shows a strong binding preference for oligoadenylated RNAs over polyadenylated RNAs. **Binds associated protein, especially Pat1, in the P-body where it facilitates the deadenylation-dependent decapping of mRNA thereby regulating mRNA decay and subsequent degradation by the 5' to 3' pathway.**

> 'I think we need to include composition and the specific function of the Lsm1-7 complex, which is to bind RNA, maybe less on the process. Note that we already have a term for the "Lsm1-7-Pat 1 complex". It seems like most of the functions you ascribe to the Lsm1-7 complex are what it does when it is bound to Pat1, so I wonder if it would be more appropriate to put the description of biological role in mRNA decay and P-body formation into that def, and put only the RNA binding part in the def of the Lsm1-7 complex term.'

Yes, looks like LSM1-7 always binds Pat1 in the P-body and then becomes functionally.

xrefs:
GOC:bhm, GOC:krc
IntAct:EBI-16419327
PMID:23221597, 23620288, 26764377 

> 'not sure I'm keen to put three A. thaliana refs on defs for a highly conserved complex, would prefer to use the review I cited above.'

Yes, and add review PMID:27627834.

Relationships:
is_a GO:0043234 protein complex
part_of GO:0000932 P-body 

> 'I'm not sure we can say this complex is ALWAYS part_of the P-body. I kind of thought that it was involved in bringing mRNAs subject to decay TO the P-body, implying that it binds them elsewhere and helps to localize them there'

Yes, you are right. So it's part_of cytoplasm.

capable_of_part_of GO:NEW positive regulation of GO:0000290 deadenylation-dependent decapping of nuclear-transcribed mRNA (#14410) 

> 'seems this might be more appropriate to say for the existing term for the "Lsm1-7-Pat1 complex"'

and then make LSM1-7 complex part_of Lsm1-7-Pat1 complex? At least it's Lsm1-7-Pat1 complex has_part LSM1-7 complex.


Name: LSM2-8 complex
Def: A protein complex that is part of the U6 snRNP spliceosome machinery. 
Revised Def: A heteroheptameric, **nuclear** protein complex composed of Lsm2, Lsm3, Lsm4, Lsm5, Lsm6, Lsm7, and Lsm8, or orthologs thereof, that selectively binds to **snRNAs, in particular** the U6 snRNA, and also to unspliced transcripts localized within the nucleus.

xrefs:
GOC:bhm, GOC:krc
IntAct:EBI-16419597
PMID:23221597, 23620288, 28482101 
> 'again, not sure I'm keen to put three A. thaliana refs on defs for a highly conserved complex, would prefer to use the review I cited above.'

Yes add other review, too, see above.

Relationships:
is_a GO:0043234 protein complex
part_of GO:0005634 nucleus
GO:0005688 U6 snRNP has_part LSM2-8 complex
capable_of_part_of GO:0000398 mRNA splicing, 
> 'via spliceosome seems like this belongs on the U6 snRNP complex' 

Yes, I agree.

capable_of GO:0017070 U6 snRNA binding 

> The Tharun et al review uses the phrase "Sm-core of U6 snRNP" to refer to the Sm ring that binds to U7, with the composition listed in the first quote from the Marzluff & Koreski paper.

"core" is so contentious: is it just the Sm-like ring protein complex or the Sm-like snRNP complex? The snRNP complex associates with more proteins and complexes to make larger machineries!

> -- Lsm family ring complex
--- Lsm1-7 complex
--- Lsm2-8 complex
--- Sm complex
--- U8 snoRNA associated Lsm complex
--- snR5 associated Lsm complex

I'd call it "Sm-like family complex" - with your suggested term as exact synonym?
Plus the existing X-Sm complexes in GO, e.g.
GO:0034715 pICln-Sm protein complex
GO:0034719 SMN-Sm protein complex
GO:0034731 Lsm-containing SMN-Sm protein complex
GO:0034730 SmD-containing SMN-Sm protein complex
GO:0032797 SMN complex
GO:0034718 SMN-Gemin2 complex
GO:0034731 Lsm-containing SMN-Sm protein complex
GO:1990726 Lsm1-7-Pat1 complex 
No idea if I found all as I just did a text search.

Phuh, I hope I commented on everything that needed commenting on!

**Comment by bmeldal** [comment]
@thomaspd 

>I think we should have a discussion about functional classifications of complexes. I would suggest that we ANNOTATE complexes with functions, complete with evidence code and reference, rather than defining their functions in the ontology. The GO terms could reflect complex membership.

This is in contrast to what we've done over the last few years. Making new terms only based on composition was getting exponentially too big so we were trying to stick to grouping terms one level above composition and aimed for functional relationships.

We are also using the functional grouping terms to create structure in the ontology. The structure below 'protein complex' was really flat before we started our project.

I was advised to add capable_of and capable_of_part_of relationships so the reasoner could then bring existing terms into new grouping terms using the inference. Otherwise we had to find all relevant exisiting terms by hand. @dosumis was instrumental in facilitating this.

This should be discussed in the WG and also get a view from the users as to what they are using.

**Comment by krchristie** [comment]
@bmeldal - It sounds like we're in agreement about the "Lsm1-7 complex" and "Lsm2-8 complex" terms.

However, regarding a grouping term, I was thinking of something specific for the Sm/Lsm rings, though that might not make sense if some/all of these rings are never "free" but only form on RNA, as you mentioned for the Sm rings that interact with most of the spliceosomal snRNAs (except U6 which binds Lsm2-8). 

It seems that you are including complexes that include additional components. If we are going to include everything that has one of these Sm/Lsm ring (sub)complexes as a part, then we should  probably also include the terms for the snRNPs that contain U1, U6, etc.

So, we should decide whether either or both of these grouping terms is appropriate. Let me know what you think.

**Comment by bmeldal** [comment]
Well, the question which type of grouping therm is useful? We did something similar for the class of  GO:0070603 SWI/SNF superfamily-type complex recently:
https://github.com/geneontology/go-ontology/issues/14106

We could use one overarching term for "Sm-like protein family complex" that include the possibility of snRNAs as members and have a child of LSM-like ring complexes that don't include the snRNAs and add a comment that the complex binds the snRNAs after assembly.

**Comment by krchristie** [comment]
So to confirm I am understanding @bmeldal, you would find it useful to have **BOTH** a term for:
- "Sm-like protein family complex" for all complexes that contain any of these Sm or Lsm rings and also other non-Lsm/Sm components
- "LSM-like `type?` ring complexes" - What would this term be for? Just the LSM complexes that assemble on their own without requiring the RNA to assemble onto?, so therefore NOT the Sm ring that binds to snRNAs other than U6? Also, seems odd to put the word "like" right after an acronym, LSM, that already includes the word "like". Perhaps "LSM-type" would be better.

**Comment by bmeldal** [comment]
We've discussed these complexes further:

The Sm/Lsm rings need associated proteins, such as Pat1 for Lsm1-7, to function but these differ dep on the specific ring and the RNA substrate. Therefore, the simple rings structure aren't really functional. Having said that, we are not planning on making all potential variants right now. 

**Lsm1-7**:
- seems Pat1 (which comes in different flavours!) is always required.
- [ ] Keep existing GO:1990726 Lsm1-7-Pat1 complex and don't make new Lsm1-7 complex term.
- We will add Pat1 (or its variants) to our complexes in the CP.

- [ ] update def:
*-Def*: A conserved protein complex that plays an important role in coupling deadenylation and decapping in the 5'-to-3' mRNA decay pathway. An example is found in S. cerevisiae.
*+Def, as discussed above for Lsm1-7*: A conserved, heteroheptameric, cytoplasmic protein complex composed of Lsm1, Lsm2, Lsm3, Lsm4, Lsm5, Lsm6, Lsm7, and Pat1, or orthologs thereof, that shows a strong binding preference for oligoadenylated RNAs over polyadenylated RNAs. May bind further associated proteins.  Facilitates the deadenylation-dependent decapping of mRNA in the P-body thereby regulating mRNA decay and subsequent degradation by the 5' to 3' pathway.

xrefs:
GOC:bhm, GOC:krc
IntAct:EBI-16419327
PMID:23221597, 27627834, 28768202, 19121818

Relationships:
part_of cytoplasm.
capable_of_part_of GO:0000290 deadenylation-dependent decapping of nuclear-transcribed mRNA (replace current capable_of_part_of term?)


**Lsm2-8**: 
I haven't identified a full list of associated proteins and it might assemble and function directly on the snRNA so I'm thinking here the 'simple' ring complex term would be needed

- [ ]  create new term

Def: A heteroheptameric, nuclear protein complex composed of Lsm2, Lsm3, Lsm4, Lsm5, Lsm6, Lsm7, and Lsm8, or orthologs thereof, that selectively binds to snRNAs, in particular the U6 snRNA, and also to unspliced transcripts localized within the nucleus.

xrefs:
GOC:bhm, GOC:krc
IntAct:EBI-16419597
PMID:23221597, 27627834, 28768202, 19121818

Relationships:
is_a GO:0043234 protein complex
part_of GO:0005634 nucleus
capable_of_part_of GO:0000398 mRNA splicing,
capable_of GO:0017070 U6 snRNA binding

- [ ] update GO:0005688 U6 snRNP **has_part** LSM2-8 complex

**New grouping term?**
- If we go with the above we can't have a term for Lsm ring protein complexes as some also have other proteins (e.g. Pat1), so the only suitable grouping term would be for the "Sm-like protein family complex". Discuss!

**Comment by krchristie** [comment]
@bmeldal - I like your suggestions for 
- just updating the definition for the existing term for the Lsm1-7-Pat1 complex
- new term for the Lsm2-8 complex

I agree with your conclusion that the only suitable grouping term would be the broader one for "Sm-like protein family complex" that includes complexes that contain other proteins in addition to the Sm/Lsm rings. So my question for you is do you want this term. If you think it is useful, I'm happy to make it.

**Comment by bmeldal** [comment]
- [ ] **GO:NEW Sm-like protein family complex**

Def: A protein complex involved in either spliceosomal snRNP assembly in the cytoplasm, pre-mRNA splicing in the nucleus and/or decapping and the 5' to 3' mRNA degradation pathway in the cytoplasm. May consist of a large assemblies of Sm proteins (Small nuclear ribonucleoproteins), SMN proteins (survival motor neuron proteins) and/or Lsm proteins (Sm-like proteins), associated factors like Gemin proteins and small-non-coding RNAs (snRNA), pre-mRNAs or mRNAs. May form supercomplexes of the SMN core complex with other Sm and associated proteins. Subcomplexes are often involved in more than one pathway and specificity is determined by their associated proteins/complexes.

xrefs:
GOC:bhm
PMID:27627834, 19121818

Relationships:
is_a GO:0032991 macromolecular complex

**Comment by krchristie** [comment]
I would prefer a definition more focused on the composition, and mentioning the roles more generally, something like this:

**Term**: GO:NEW Sm-like protein family complex
**Def**: A protein complex containing members of the Like-Sm family of proteins, which includes both the Sm proteins and the Lsm proteins, and which generally form hexameric or heptameric ring structures which bind to RNA. While some of these ring complexes may form independently of RNA, many only form in association with their target RNA. In addition to Lsm-family proteins, many of these complexes contain additional protein members. Members of this family of complexes include the snRNPs which comprise the majority of the spliceosome. Others are involved in the 5' to 3' degradation pathways of mRNAs in the cytoplasm and of unspliced transcripts in the nucleus, as well as other diverse roles.
**Def xrefs**: GOC:bhm, GOC:krc, PMID:27627834, PMID:19121818
**Relationships**: is_a GO:0032991 macromolecular complex

**Comment by thomaspd** [comment]
I agree that the GO classes should focus on composition, not function.  As I'd suggested earlier, IntAct can annotate complexes with GO MF and BP terms, using evidence codes and references.  These annotations could then be used on the IntAct site to browse or search by function.  Birgit, would that work for you?

**Comment by krchristie** [comment]
@thomaspd - While I am in favor of focusing on the composition as the definition of what is common in this family of complexes, I am not in favor of dropping function out of definitions entirely and leaving the functions to be defined only by annotation, especially not for specific complexes. 

Not only is this inconsistent long defined GOC practice, I think it will be detrimental to consistency in annotation. For curators to find the right terms, definitions have to be sufficiently explanatory for us to identify which is the correct term.

**Comment by thomaspd** [comment]
Yes, it's fine to have reference to function in the text definition, just not in the logical definition.

**Comment by krchristie** [comment]
I think whether or not it is OK to have it in the logical definition depends on the specificity of the complex. In this grouping term, there is clearly no logical definition that is appropriate.

**Comment by krchristie** [comment]
I have 
- put in the new terms for Lsm2-8 and the Sm-like protein family complex grouping term
- modified the definition of the existing Lsm1-7-Pat1 complex term
- given the various existing terms that contain Sm or Lsm proteins is_a relationship to the new grouping term

@bmeldal - For future, it would be really helpful if you would list the IntAct **database cross references** separately from the **definition references**, since the IntAct references belong in a different place. 

+id: GO:0120115
+name: Lsm2-8 complex
+namespace: cellular_component
+def: "A heteroheptameric, nuclear protein complex composed of Lsm2, Lsm3, Lsm4, Lsm5, Lsm6, Lsm7, and Lsm8, or orthologs thereof, that selectively binds to snRNAs, in particular the U6 snRNA, and also to unspliced transcripts localized within the nucleus." [**GOC:bhm, GOC:krc, PMID:19121818, PMID:23221597, PMID:27627834, PMID:28768202**]
+**xref: IntAct:EBI-16419597**
+is_a: GO:0120114 ! Sm-like protein family complex
+created_by: kchris
+creation_date: 2017-11-06T18:56:22Z

**Comment by bmeldal** [comment]
Thanks @krchristie 

Ok, will separate the def xref and dbxrefs in future.

As for the def: did you drop the SMN proteins (survival motor neuron proteins) on purpose? To me they seemed like a third group within the Sm-like family of proteins.

**Comment by krchristie** [comment]
Hi @bmeldal - Regarding the def, I wasn't aware that the SMN proteins were considered to be members of the Lsm family. Looking back through the Tharun review:
> Tharun S. Roles of eukaryotic Lsm proteins in the regulation of mRNA function.
Int Rev Cell Mol Biol. 2009;272:149-89. doi: 10.1016/S1937-6448(08)01604-3.
Review.  PMID:19121818

to see if I missed anything, I don't see any mention of the SMN proteins as a member of the Lsm family. The only occurance of "SMN" at all in the Tharun review is this citation:
> R.S. Pillai, M. Grimmler, G. Meister, C.L. Will, R. Luhrmann, U. Fischer, D. Schumperli
Unique Sm core structure of U7 snRNPs: Assembly by a specialized SMN complex and the role of a new component, Lsm11, in histone RNA processing
Genes Dev., 17 (2003), pp. 2321-2333

The Pillai et al. paper talks about the SMN complex as an assembly factor for Sm proteins, but does not refer to SMN as an Sm protein itself.

In addition, this review about SMN:
> Raimer AC, Gray KM, Matera AG. SMN - A chaperone for nuclear RNP social
occasions? RNA Biol. 2017 Jun 3;14(6):701-711. doi:
10.1080/15476286.2016.1236168. Epub 2016 Sep 20. PubMed PMID: 27648855

also only talks about SMN as a chaperone or assembly factor for Sm proteins, but not as an Sm protein itself.

So, while I agree that the SMN complex terms belong as types of the 'Sm-like protein family complex' term, I do not see any indication that SMN is itself a member of the Lsm family.

Do you have papers indicating that SMN is a member of the Lsm family?

**Comment by bmeldal** [comment]
Sorry, I went by the rather short GO term defs and somehow assumed they were all part of the same family. I just looked at SMN directly and you are right, they are a chaperones/assembly proteins and the SMN complex is responsible for the assembly of the Sm/Lsm rings. The SMN complex itself shouldn't even be part of this complex group.

Sorry, @krchristie I didn't read up on the SMN separately.

**Comment by krchristie** [comment]
@bmeldal - Thanks for checking on this. To be clear before I fix this, I'm thinking it's OK for the SMN-Sm protein complex to stay an is_a child of the grouping term. Here are the current child terms of the grouping term:

**NOT a is_a 'Sm-like protein family complex'**
- GO:0032797 SMN complex
- GO:0034718 SMN-Gemin2 complex

**?? Are these is_a 'Sm-like protein family complex??'** - they do contain Lsm-family proteins
- GO:0034719 SMN-Sm protein complex
- and two child terms:
1. - GO:0034731 Lsm-containing SMN-Sm protein complex
2. - GO:0034730 SmD-containing SMN-Sm protein complex

**Definitely is_a 'Sm-like protein family complex'**
- GO:1990726 Lsm1-7-Pat1 complex
- GO:0034715 pICln-Sm protein complex
- Also, need to add this parentage for all the snRNPs that become part of the spliceosome, except for U6 snRNP which I've already done

**Comment by bmeldal** [comment]
Hi @krchristie yes, I think you are right with the assessment above, plus of course the new Lsm2-8 term but that is already correctly assigned.

>Also, need to add this parentage [Sm-like protein family complex] for all the snRNPs that become part of the spliceosome.

Wow, I hadn't thought of this, but yes, there are more children!

**Comment by krchristie** [comment]
I have removed this relationship:
GO:0034719 SMN-Sm protein complex
**is_a** 'Sm-like protein family complex'

and added this one:
'small nuclear ribonucleoprotein complex'
**is_a** 'Sm-like protein family complex'

This term is the parent term of all the splicesosomal snRNPs and also the U7 snRNP, which is also a Sm-family complex according to the Tharun review:

> Another Sm-like protein complex conserved in higher eukaryotes and involved in a specialized mechanism through which metazoan histone mRNA 3'-ends are generated is the unique Sm-core
complex of the U7 snRNP, which is made up of five Sm proteins and two Lsm proteins.

I think this covers everything we've discussed, but let me know if you spot anything we missed, @bmeldal

**Comment by bmeldal** [comment]
Thank you, Karen, your efforts are much appreciated! Complexes will be in the next release (just missed the one from yesterday).
